apps/src/JSInterpreter.js

```diff
@@ -828,7 +828,7 @@ JSInterpreter.prototype.getNearestUserCodeLine = function () {
   }
   var userCodeRow = -1;
   for (var i = 0; i < this.interpreter.stateStack.length; i++) {
-    var node = this.interpreter.stateStack[i].node;
+    var node = this.interpreter.stateStack[this.interpreter.stateStack.length - 1 - i].node;
```

Comment:
Why this instead of `for (let i = this.interpreter.stateStack.length - 1; i >= 0; i--)`?